person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || is_of_age?
+  end
+
+  def validate_name
+    @name = 'Unknown' if @name.nil?
+  end
+
+  def validate_age
+    @age = 0 if @age.negative?
+  end
+
+  def of_age?
```

Comment:
- [ ] Please fix the typo. It should be `is_of_age?` 😄

Reply:
here is the error i get when the linter runs
```
C: Naming/PredicateName: Rename is_of_age? to of_age?.
  def is_of_age?
      ^^^^^^^^^^
```

Reply:
Fixed.